dpimport/database/__init__.py

```diff
@@ -68,8 +68,7 @@ def exists(self, probe):
         :type probe: dict
         '''
         doc = self.db.toc.find_one({
-            'path': probe['path'],
-            'size': probe['size']
+            'content': probe['content'],
```

Comment:
If a single digit of a row in an existing document changes, particularly those that indicate data availability bit (1/0), the size of the document remains same. `import.py` will skip re-import of that document by the above logic. But the skipping would not be right.